facets.js

```diff
@@ -154,20 +154,26 @@ export const facets = {
    * - the type of the referrer, e.g. `search`
    * - the vendor of the referrer, regardless of type, e.g. `*:google`
    */
-  enterSource: (bundle) => bundle.events
-    .filter((evt) => evt.checkpoint === 'enter')
-    .map((evt) => evt.source)
-    .filter((source) => source)
-    .map((source) => {
-      const referrerClass = classifyReferrer(source);
-      return referrerClass ? [
-        source,
-        `${referrerClass.type}:${referrerClass.vendor}`,
-        referrerClass.type,
-        `*:${referrerClass.vendor}`,
-      ] : source;
-    })
-    .flat(),
+  enterSource: (bundle) => {
+    const normalizedSources = bundle.events
+      .filter((evt) => evt.checkpoint === 'enter')
+      .map((evt) => evt.source)
+      .filter((source) => source)
```

Comment:
```suggestion
      .filter((source) => source)
      .map((source) => source.replace(/\/#$/, '')
```

that's how I'd do it.

Reply:
updated.